ouroboros-consensus-cardano/src/tools/Cardano/Tools/DBTruncater/Types.hs

```diff
@@ -0,0 +1,14 @@
+module Cardano.Tools.DBTruncater.Types (module Cardano.Tools.DBTruncater.Types) where
+
+import           Cardano.Tools.DBAnalyser.Types
+import           Ouroboros.Consensus.Block.Abstract
+
+data DBTruncaterConfig = DBTruncaterConfig {
+    dbDir         :: FilePath
+  , truncatePoint :: TruncatePoint
+  , blockType     :: BlockType
+  , verbose       :: Bool
+  }
+
+newtype TruncatePoint = TruncatePoint SlotNo
```

Comment:
`Point` is a fundamental type in our database. I recommend trying to avoid use of the word "point" when unless you're actually talking about `Point`.

Hence I would suggest renaming this type to `TruncationSlot`.